homeassistant/components/matter/binary_sensor.py

```diff
@@ -161,4 +161,100 @@ def _update_from_device(self) -> None:
         entity_class=MatterBinarySensor,
         required_attributes=(clusters.DoorLock.Attributes.DoorState,),
     ),
+    MatterDiscoverySchema(
+        platform=Platform.BINARY_SENSOR,
+        entity_description=MatterBinarySensorEntityDescription(
+            key="SmokeCoAlarmDeviceMutedSensor",
+            measurement_to_ha=lambda x: x
+            == clusters.SmokeCoAlarm.Enums.MuteStateEnum.kMuted,
```

Comment:
I think it would be nice if we wrap the lambdas in parenthesis because now the second part is on the same level as the rest of the properties